.gitignore

```
@@ -2,3 +2,4 @@
 .bundle
 .netlify
 _site
+**/.DS_Store
```

```suggestion
.DS_Store
```

This is a file right? Just the filename should catch it at any level of the repo.